### PR TITLE
Update newsletter blog posts

### PR DESCRIPTION
**Pull request by thc202**
Add dash before the year in the titles.
Remove draft state and TODO tag.
Remove unnecessary empty lines.
Address typos.
Update links to site and source code.
Fix links that were split into multiple lines.
Fix image tag.
Use HTTPS when available and use latest URL (when redirected).
Update zapbot image.
Add code tags to example URLs (and related nodes, for consistency) to
not be rendered as link.
Restore missing content.

### CODE REVIEW

File: site/content/blog/2015-11-02-zap-newsletter-2015-november/index.md [hunk 1]
```diff
@@ -77,28 +74,27 @@ There are 2 new options:
...
-      * company3 
-        * GET:p1(ddd) 
+  * `https://www.example.com`
+    * `app`
+      * `company1`
+        * `GET:p1(ddd)`
+      * `company2`
+        * `GET:p1(ddd)`
+      * `company3`
+        * `GET:p1(ddd)`
```
Comment (kingthorin) [inline site/content/blog/2015-11-02-zap-newsletter-2015-november/index.md]: Would it be better to do this as a single block with indenting and no bullets? (console or text as the hint)

Comment (kingthorin) [reply site/content/blog/2015-11-02-zap-newsletter-2015-november/index.md]: There are a few other places too

Comment (thc202) [reply site/content/blog/2015-11-02-zap-newsletter-2015-november/index.md]: I think with bullets makes it more like the Sites tree, but happy to change.

Comment (kingthorin) [reply site/content/blog/2015-11-02-zap-newsletter-2015-november/index.md]: That's fair.

**Pull request merged**